Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -7695,8 +7697,13 @@ - (BOOL)supportsTextSelectionWithCharacterGranularity
 #if ENABLE(REVEAL)
 - (void)requestRVItemInSelectedRangeWithCompletionHandler:(void(^)(RVItem *item))completionHandler
 {
-    _page->requestRVItemInCurrentSelectedRange([revealItemSelectionHandler = makeBlockPtr(completionHandler)](const WebKit::RevealItem& item) {
-        revealItemSelectionHandler(item.item());
+    _page->requestRVItemInCurrentSelectedRange([completionHandler = makeBlockPtr(completionHandler), weakSelf = WeakObjCPtr<WKContentView>(self)](const WebKit::RevealItem& item) {
+        auto strongSelf = weakSelf.get();
+        if (!strongSelf)
+            return completionHandler(nil);
+
+        SetForScope editMenuPresentationScope { strongSelf->_isStartingToPresentEditMenu, YES };
```

Comment:
isStartingToPresentEditMenu doesn't sound quite right but I don't have a better suggestion.

Reply:
Yeah, it's difficult to come up with a name for this one. I do think "edit menu" is the correct term that refers to the right click edit menu and (what was previously referred to as) the callout bar, but maybe the `isStartingToPresent` part is weird.

Perhaps, something like `_shouldAllowSelectAllAction` that instead describes how the flag is utilized?

(Also, I'll need to move the flag in order to fix the watchOS/tvOS builds)

Reply:
Hm... maybe something like _isPresentingEditMenu or _isPreparingEditMenu might make sense?

Reply:
Sounds good to me — I'll go with `_isPreparingEditMenu`.